cookbooks/web/attributes/default.rb

```diff
@@ -1,3 +1,6 @@
 default[:web][:base_directory] = "/srv/www.openstreetmap.org"
 default[:web][:pid_directory] = "/var/run/web"
 default[:web][:log_directory] = "/var/log/web"
+
+# need the APT source for cgimap to be able to install the package
+default[:apt][:sources] = node[:apt][:sources] | ["openstreetmap-cgimap"]
```

Comment:
Another example, `logstash-forwarder`, includes `[:apt][:sources] = ["logstash-forwarder"]` in its default attributes in `roles/logstash-forwarder.rb`. Is it safe to do that in `web-backend.rb`, or will one or other set of default attributes override the other, rather than adding together?